### ISSUE TITLE
could not get host score estimate: cannot call /host/estimatescore without the renter module

### ISSUE THREAD

**Issue by poiuty** [NebulousLabs/Sia#2218]
Start siad with flag `-M gctwh`, change price.
```
# siac host | grep Price
	Price:        150 SC / TB / Month

# siac host config minstorageprice 100SC
could not get host score estimate: cannot call /host/estimatescore without the renter module

# siac host | grep Price
	Price:        100 SC / TB / Month

# siac version
Sia Client v1.3.0
Sia Daemon v1.3.0
```

**Comment by pY4x3g** [comment]
same issue here

**Comment by dxiri** [comment]
same problem, I tried changing the collateralbudget:

```
./siac host config collateralbudget 3KS
could not get host score estimate: cannot call /host/estimatescore without the renter module
```

**Comment by walec51** [comment]
same problem, do devs even care?

**Comment by avahowell** [comment]
should be fixed after #2323 is merged. Note that your commands are actually succeeding (your host settings are being successfully updated), and this error can be safely ignored.

**Comment by walec51** [comment]
ok, thank you for the reply
